utils/sampler/uniform_test.go

```diff
@@ -4,78 +4,52 @@
 package sampler
 
 import (
-	"fmt"
 	"math"
 	"slices"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 )
 
-var (
-	uniformSamplers = []struct {
-		name    string
-		sampler Uniform
-	}{
-		{
-			name: "replacer",
-			sampler: &uniformReplacer{
-				rng: globalRNG,
-			},
-		},
-		{
-			name: "resampler",
-			sampler: &uniformResample{
-				rng: globalRNG,
-			},
-		},
-		{
-			name:    "best",
-			sampler: NewBestUniform(30),
-		},
-	}
-	uniformTests = []struct {
-		name string
-		test func(*testing.T, Uniform)
-	}{
-		{
-			name: "can sample large values",
-			test: UniformInitializeMaxUint64Test,
-		},
-		{
-			name: "out of range",
-			test: UniformOutOfRangeTest,
-		},
-		{
-			name: "empty",
-			test: UniformEmptyTest,
-		},
-		{
-			name: "singleton",
-			test: UniformSingletonTest,
-		},
-		{
-			name: "distribution",
-			test: UniformDistributionTest,
-		},
-		{
-			name: "over sample",
-			test: UniformOverSampleTest,
-		},
-		{
-			name: "lazily sample",
-			test: UniformLazilySample,
-		},
-	}
-)
+var uniformTests = []struct {
+	name string
+	test func(*testing.T, Uniform)
+}{
+	{
+		name: "can sample large values",
+		test: UniformInitializeMaxUint64Test,
+	},
+	{
+		name: "out of range",
+		test: UniformOutOfRangeTest,
+	},
+	{
+		name: "empty",
+		test: UniformEmptyTest,
+	},
+	{
+		name: "singleton",
+		test: UniformSingletonTest,
+	},
+	{
+		name: "distribution",
+		test: UniformDistributionTest,
+	},
+	{
+		name: "over sample",
+		test: UniformOverSampleTest,
+	},
+	{
+		name: "lazily sample",
+		test: UniformLazilySample,
+	},
+}
 
-func TestAllUniform(t *testing.T) {
-	for _, s := range uniformSamplers {
-		for _, test := range uniformTests {
-			t.Run(fmt.Sprintf("sampler %s test %s", s.name, test.name), func(t *testing.T) {
-				test.test(t, s.sampler)
-			})
-		}
+func TestUniform(t *testing.T) {
+	for _, test := range uniformTests {
+		t.Run(test.name, func(t *testing.T) {
+			test.test(t, NewUniform())
```

Comment:
Because we only have one sampler now - I'm not sure that this makes sense as being a test vector.